diff --git a/proton-bridge/internal/frontend/bridge-gui/bridge-gui/build.sh b/proton-bridge/internal/frontend/bridge-gui/bridge-gui/build.sh
index c299d5ce..614f8a90 100755
--- a/proton-bridge/internal/frontend/bridge-gui/bridge-gui/build.sh
+++ b/proton-bridge/internal/frontend/bridge-gui/bridge-gui/build.sh
@@ -77,7 +77,10 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
     check_exit "Failed installing gRPC for macOS / Apple Silicon"
     ${VCPKG_EXE} install sentry-native:x64-osx-min-10-15 grpc:x64-osx-min-10-15 --overlay-triplets=vcpkg/triplets --clean-after-build
     check_exit "Failed installing gRPC for macOS / Intel x64"
-elif [[ "$OSTYPE" == "linux"* ]]; then
+elif [[ "$OSTYPE" == "linux"* && "$(uname -m)" == "aarch64" ]]; then
+    ${VCPKG_EXE} install sentry-native:arm64-linux grpc:arm64-linux
+    check_exit "Failed installing gRPC for Linux / ARM64"
+elif [[ "$OSTYPE" == "linux"* && "$(uname -m)" == "x86_64" ]]; then
     ${VCPKG_EXE} install sentry-native:x64-linux grpc:x64-linux --clean-after-build
     check_exit "Failed installing gRPC for Linux / Intel x64"
 else
