diff --git a/proton-bridge/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake b/proton-bridge/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
index 10e1ea16..61d5d6b1 100644
# Update to the icu version qt 6.7.0 is linked against (73) 
# and add the missing libQt6QuickControls2Basic.so.6 and libQt6QuickControls2BasicStyleImpl.so.6
# else Notifications make the app crash
--- a/proton-bridge/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
+++ b/proton-bridge/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
@@ -45,6 +45,7 @@ endmacro()
 
 #Qt6
 AppendQt6Lib("libQt6QuickControls2.so.6")
+AppendQt6Lib("libQt6QuickControls2Basic.so.6")
 AppendQt6Lib("libQt6Quick.so.6")
 AppendQt6Lib("libQt6QmlModels.so.6")
 AppendQt6Lib("libQt6Qml.so.6")
@@ -54,9 +55,9 @@ AppendQt6Lib("libQt6Gui.so.6")
 AppendQt6Lib("libQt6Core.so.6")
 AppendQt6Lib("libQt6QuickTemplates2.so.6")
 AppendQt6Lib("libQt6DBus.so.6")
-AppendQt6Lib("libicui18n.so.56")
-AppendQt6Lib("libicuuc.so.56")
-AppendQt6Lib("libicudata.so.56")
+AppendQt6Lib("libicui18n.so.73")
+AppendQt6Lib("libicuuc.so.73")
+AppendQt6Lib("libicudata.so.73")
 AppendQt6Lib("libQt6XcbQpa.so.6")
 AppendQt6Lib("libQt6WaylandClient.so.6")
 AppendQt6Lib("libQt6WlShellIntegration.so.6")
@@ -72,6 +73,7 @@ AppendQt6Lib("libQt6QuickWidgets.so.6")
 # QML dependencies
 AppendQt6Lib("libQt6QmlWorkerScript.so.6")
 AppendQt6Lib("libQt6Widgets.so.6")
+AppendQt6Lib("libQt6QuickControls2BasicStyleImpl.so.6")
 AppendQt6Lib("libQt6QuickControls2Impl.so.6")
 AppendQt6Lib("libQt6QuickLayouts.so.6")
 AppendQt6Lib("libQt6QuickDialogs2.so.6")
